Makefile

```diff
@@ -64,7 +64,8 @@ package: build-release
 
 check-wasm = $(cargo) check --target wasm32-unknown-unknown --manifest-path $(wasm)/Cargo.toml
 check:
-	$(cargo) check && \
+	$(cargo) check --workspace --exclude namada_benchmarks && \
+	$(cargo) +$(nightly) check --benches && \
```

Comment:
does this need nightly?

Reply:
I tried to remove it but it complains about not being able to compile `namada_tests`

Reply:
we'll be able to use stable after https://github.com/anoma/namada/pull/1771